### DISCUSSION TITLE
Intent 8 analysis

### DISCUSSION THREAD

**Discussion by Kevsy** [camaraproject/EdgeCloud#179] [General]
### Developer intents: Provisioning intents

| Intent 8 | 5SED |5MEE |EXRA|EXRC| EXCA | EXSM|  GMEC |
| -------- |------| ----|----| ---|------|-----|-------|
| 8. “I can ask the operator to provide the artifacts details for already stored artifacts”   | No           |No    | ?          | ?     | ?            | ?     | Yes   |

| GMEC         | Details       |Comments|
| --------------|-------------| -----------|
| (to complete) | (to complete) | (to complete) |

(others to complete as appropriate  as appropriate)

**Comment by gunjald** [comment]
The application artefacts management APIs provide the GET method to retrieve the details of the already submitted artefacts which an application provider may want to query.
